src/terrain/services/teams.clj

```diff
@@ -36,9 +37,15 @@
 (defn join [{user :shortUsername} name]
   (ipg/join-team user name))
 
-(defn join-request [{user :shortUsername user-name :commonName email :email :as user-info} name message]
+(defn join-request [{user :shortUsername user-name :commonName email :email} name message]
   (let [admin (first (ipg/get-team-admins user name))]
     (cn/send-team-join-notification user-name email name admin message)))
 
+(defn deny-join-request [{user :shortUsername} name requester message]
+  (ipg/verify-team-exists user name)
```

Comment:
(Sorry if this is obvious...) So if the user hits this endpoint and the team doesn't exist, will the UI actually get a 404 back?  Is there an error message that lets the user know the team doesn't exist?